src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -120,19 +250,73 @@ private void initMenubar() {
             a.setTitle("Confirm Exit");
             a.setHeaderText("Do you really want to exit?");
             Optional<ButtonType> result = a.showAndWait();
-            if (result.get() == ButtonType.OK) {
-                Platform.exit();
-                System.exit(0);
+            if (result.isPresent()) {
+                if (result.get() == ButtonType.OK) {
+                    Platform.exit();
+                    System.exit(0);
+                }
+                if (result.get() == ButtonType.CANCEL) {
+                    a.close();
+                }
             }
+        });
 
-            if (result.get() == ButtonType.CANCEL) {
-                a.close();
+        btnAbout.setOnAction(event -> {
+            Alert alert = new Alert(Alert.AlertType.INFORMATION);
+            alert.setTitle("About");
+            alert.setHeaderText(null);
+            alert.setResizable(true);
+            alert.getDialogPane().setMinWidth(ABOUT_MIN_WIDTH);
+            alert.setContentText("This application is made by Contextproject group Desoxyribonucleïnezuur:\n\n"
+                    + "Ivo Wilms \n" + "Iwan Hoogenboom \n" + "Martijn van Meerten \n" + "Toine Hartman\n"
+                    + "Yannick Haveman");
+            alert.show();
+        });
+
+        btnInstructions.setOnAction(event -> {
+            Alert alert = new Alert(Alert.AlertType.INFORMATION);
+            alert.setTitle("Instructions");
+            alert.setHeaderText(null);
+            alert.setResizable(true);
+            alert.getDialogPane().setMinWidth(INSTRUCTIONS_MIN_WIDTH);
+            alert.setContentText(informationText);
+            alert.show();
+        });
+    }
+
+    /**
+     * Initializes the bookmark buttons in the menu.
+     */
+    private void initBookmarkMenu() {
+
+        btnBookmarks.setOnAction(event -> {
+            try {
+
+                FXMLLoader loader = new FXMLLoader(ProgrammingLife.class.getResource("/LoadBookmarkWindow.fxml"));
+                AnchorPane page = loader.load();
+                GuiLoadBookmarkController gc = loader.getController();
+                gc.setGraphController(graphController);
+                gc.initColumns();
+                Scene scene = new Scene(page);
+                Stage bookmarkDialogStage = new Stage();
+                bookmarkDialogStage.setScene(scene);
+                bookmarkDialogStage.setTitle("Load Bookmark");
+                bookmarkDialogStage.initOwner(ProgrammingLife.getStage());
+                bookmarkDialogStage.showAndWait();
+
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Actually handle this error (give a message to the user, or, better, show the option red in the menu with an explanation)